### PR TITLE
fix(autoware_pointcloud_preprocessor): launch file load parameter from yaml

### PR DESCRIPTION
**Pull request by vividf**
## Description
PR https://github.com/autowarefoundation/autoware.universe/pull/8509, https://github.com/autowarefoundation/autoware.universe/pull/8300, and https://github.com/autowarefoundation/autoware.universe/pull/8466 replace default parameters by using parameter schema and parameter yaml file in the pointcloud preprocessor.

This PR enables the launcher to load the parameter from yaml file.

Note that this PR should be merged after all of the PRs mentioned above are merged.

## Related links

**Parent Issue:**

- Link

## How was this PR tested?

```
ros2 launch autoware_pointcloud_preprocessor preprocessor.launch.xml
ros2 launch autoware_pointcloud_preprocessor polygon_remover.launch.py 
```

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py
Comment (YoshiRi) [inline sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py:1]: [imho]
I think it's better to use `launch_arguments.append(DeclareLaunchArgument(name, default_value=default_value))` to enable launch args so that user can give parameter file from outside of this launcher. (But you can do this in another PR.)

**Review by YoshiRi — COMMENTED** [review]
LGTM( only looking into the code, did not run evaluator yet)

**Comment by vividf** [comment]
1. https://github.com/autowarefoundation/autoware.universe/pull/8469
2. https://github.com/autowarefoundation/autoware.universe/pull/8509
3. https://github.com/autowarefoundation/autoware.universe/pull/8300
4. https://github.com/autowarefoundation/autoware.universe/pull/8466
This PR needs to wait for the above PRs be merged

File: sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py
Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py:1]: Sorry for the late reply!

I don't really understand which line are you mentioning, is it line 171?

Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py:1]: I understand your concern for consistency.
However, using that makes the launcher more complex than before (and 15 -20 lines more)

I would like to keep it for this PR. If it is really need it, let me work on it on another PR :) 

Thanks!

Comment (YoshiRi) [reply sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py:1]: > If it is really needed, let me work on it on another PR :)

I agree! Thanks for your reply!

**Review by YoshiRi — APPROVED** [review]
